.github/scripts/release.sh

```
@@ -122,172 +119,282 @@ if [ ! -x "$UV_BIN" ]; then
   exit 1
 fi
 
-# Determine target branch and default branch
-DEFAULT_BRANCH=$(git remote show origin | grep 'HEAD branch' | cut -d' ' -f5)
-if [ -z "$DEFAULT_BRANCH" ]; then
-  printf "%b[ERROR] Could not determine default branch from remote%b\n" "$RED" "$RESET"
-  exit 1
-fi
+# Helper function to prompt user to continue
+prompt_continue() {
+  local message="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)